EXM_130/Makefile

```diff
@@ -19,7 +19,6 @@ info:
 connectathon:
 	$(info connectathon checks out a specific commit SHA in case filepaths are updated)
```

Comment:
Might want to remove this line since we're getting rid of the SHA

Reply:
Good catch!